src/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "storage-init-errors-shield-study",
+  "description": "A tiny Shield Study which checks that QuotaManagerService/IndexedDB are initializing successfully",
+  "version": "1.0.0",
+  "manifest_version": 2,
+  "applications": {
+    "gecko": {
+      "id": "storage-init-errors-shield-study@shield.mozilla.org",
+      "strict_min_version": "63.0"
+    }
+  },
+  "experiment_apis": {
+    "study": {
+      "schema": "./privileged/study/schema.json",
+      "parent": {
+        "scopes": ["addon_parent"],
+        "script": "./privileged/study/api.js",
+        "paths": [["study"]]
+      }
+    },
+    "testIDBOpen": {
+      "schema": "./privileged/testIDBOpen/schema.json",
+      "parent": {
+        "scopes": ["addon_parent"],
+        "script": "./privileged/testIDBOpen/api.js",
+        "paths": [["testIDBOpen"]]
+      }
+    }
+  },
+  "permissions": ["storage", "alarms"],
+  "background": {
+    "scripts": ["studySetup.js", "background.js"]
+  },
+  "icons": {
+    "48": "icons/shield-icon.svg"
```

Comment:
Hm, where does this icon show up? Not sure it's important but just curious, do we still show these in `about:addons` when installed via Shield/Normandy?

Reply:
To be honest I leaved it there mostly because I wasn't sure if the shield icon was a convention that we use for the shield experiments or a part of the "shield study minimal example" that can be just removed.

It is visible for sure in about:debugging, I didn't checked if we explicitly hide addons installed from shield/normandy in about:addons (but I kind of recall that from time to time I saw a shield/normandy related addon briefly appearing in about:addons).